tsconfig.json

```
@@ -3,7 +3,7 @@
     "sourceMap": true,
     "noImplicitAny": true,
     "module": "es2020",
-    "target": "es6",
+    "target": "es2019",
```

Based on [browser support here](http://kangax.github.io/compat-table/es2016plus/), this actually can be increased to 
```suggestion
    "target": "es2021",
```

Let me run a quick check.

---

I think browsers have no problem even with es2022, but Terser breaks. This is the highest value that lets the build complete:
```suggestion
    "target": "es2020",
```